stubs/CoreGenericAttributes.phpstub

```diff
@@ -0,0 +1,19 @@
+<?php
+
+#[Attribute(Attribute::TARGET_METHOD)]
+final class ReturnTypeWillChange
```

Comment:
https://github.com/vimeo/psalm/blob/eb64a471c4eba7886d59703695853591335aee2f/stubs/CoreGenericClasses.phpstub#L529

ReturnTypeWillChange is already here. Probably because we need to use it before php 8